evap/evaluation/models.py

```diff
@@ -470,7 +470,10 @@ def evaluation_ends_soon(self):
 
     @property
     def days_until_evaluation(self):
-        return (self.vote_start_datetime.date() - date.today()).days
+        days_left = (self.vote_start_datetime.date() - date.today()).days
+        if self.vote_start_datetime < datetime.now():
+            days_left -= 1
+        return days_left
```

Comment:
there's a `if course.days_until_evaluation == 0 and state != 'in_evaluation'` somewhere in the template, i think the second part can be removed, please check that.